content/departments/engineering/dev/tools/scaletesting.md

```diff
@@ -156,10 +166,15 @@ The following repositories are available to test against repositories with a mas
 
 ### Perforce
 
+As small tool named [Synthforce](https://github.com/sourcegraph/synthforce) has been created to generated synthetic repos (depots) in Perforce. For more details on how to use the tool to generate more synthetic files or history on a particular depot please see the [README](https://github.com/sourcegraph/synthforce/blob/main/README.md).
+
 #### Large depot
 
 WIP See `RacoonTest` (name to be changed) over https://github.com/sourcegraph/sourcegraph/issues/42091
```

Comment:
This has been changed to a boring corporate-proof name now right?

Reply:
Aye it has. We can no longer rely on the spice 🌶️ to tell us the repo size